src/time_machine/__init__.py

```diff
@@ -498,7 +499,19 @@ def __init__(self) -> None:
         self.time = _EscapeHatchTime()
 
     def is_travelling(self) -> bool:
-        return bool(coordinates_stack)
+        return bool(traveller_stack)
 
 
 escape_hatch = _EscapeHatch()
+
+
+def __getattr__(name: str) -> type:
+    if name == "Coordinates":
+        warnings.warn(
+            "The 'Coordinates' class has been renamed to 'Traveller' and "
+            "the alias will be removed in a future version. ",
+            DeprecationWarning,
+            stacklevel=2,
+        )
+        return Traveller
+    raise AttributeError(f"module '{__name__}' has no attribute '{name}'")
```

Comment:
[opinion] This is a bit of a toss up, I would remove it, I can't imagine a lot of people using this, and adds a clean up task for the project.

Reply:
yes remove it